src/init/resolve.ts

```diff
@@ -37,7 +37,10 @@ export const getTemplateUrl = async (input: string): Promise<string> => {
   input = input.includes('/') ? input : `${config.official}/${input}`
   input = input.includes('#') ? input : `${input}#${config.branch}`
 
-  const [owner, name, branch] = input.split(/\/|#/)
+  const regex: RegExp = /(?<owner>[\w-]+)\/(?<name>[\w-]+)#(?<branch>[\w-/]+)/mg
```

Comment:
Please confirm `mg` regex flags is necessary

Reply:
I switched to a better implementation